src/core/core_timing_util.cpp

```diff
@@ -49,6 +49,11 @@ s64 nsToCycles(std::chrono::nanoseconds ns) {
     return (BASE_CLOCK_RATE * ns.count()) / 1000000000;
 }
 
+u64 nsToClockCycles(std::chrono::nanoseconds ns) {
+    const u128 temporal = Common::Multiply64Into128(ns.count(), CNTFREQ);
```

Comment:
```suggestion
    const u128 temp = Common::Multiply64Into128(ns.count(), CNTFREQ);
```

`temporal` != `temporary` in English. They mean different things. A better name here would be `temp` or `temporary`